rai/node/node.cpp

```diff
@@ -3582,7 +3583,7 @@ void rai::active_transactions::announce_votes ()
 						}
 					}
 				}
-				if (!reps->empty () && node.online_reps.online_stake () != node.config.online_weight_minimum.number ())
+				if (!reps->empty () && node.config.online_weight_minimum.number () < total_weight)
```

Comment:
Is this test correct ?

Reply:
If total_weight > online_weight_minimum from config

Reply:
Okay, that's easier to parse -- seems good !

Reply:
Yeah, perhaps I should replace it. Opposite comparison seems counterintuitive